### ISSUE TITLE
Bug: Error when deactivating device

### ISSUE THREAD

**Issue by renceInbox** [xtrinch/fcm-django#150]
Thanks for the hardwork on this library! I encountered an issue after the latest update.

Probably related to this PR #149 

Steps on how to reproduce:
- Register multiple devices for a user. We should be able to run `AbstractFCMDevice._deactivate_device_on_error_result` 
- send notification

Error
```
File "/myapp/main/notifications/engines/firebase.py", line 15, in send_notification
    device.send_message(**self.notification.notification_data)
  File "/usr/local/lib/python3.8/site-packages/fcm_django/models.py", line 204, in send_message
    self._deactivate_device_on_error_result(result)
  File "/usr/local/lib/python3.8/site-packages/fcm_django/models.py", line 245, in _deactivate_device_on_error_result
    device = self.objects.filter(registration_id=self.registration_id)
  File "/usr/local/lib/python3.8/site-packages/django/db/models/manager.py", line 179, in __get__
    raise AttributeError("Manager isn't accessible via %s instances" % cls.__name__)
AttributeError: Manager isn't accessible via FCMDevice instances
```

Version
```
fcm-django==0.3.8
```

**Comment by renceInbox** [comment]
Another way to replicate the bug

```
Python 3.8.9 (default, Apr 10 2021, 15:55:09) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
(InteractiveConsole)
>>> from fcm_django.models import FCMDevice
>>> device = FCMDevice.objects.first()
>>> device.objects
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/code.py", line 90, in runcode
    exec(code, self.locals)
  File "<console>", line 1, in <module>
  File "/usr/local/lib/python3.8/site-packages/django/db/models/manager.py", line 179, in __get__
    raise AttributeError("Manager isn't accessible via %s instances" % cls.__name__)
AttributeError: Manager isn't accessible via FCMDevice instances
>>> 
```

**Comment by renceInbox** [comment]
Another way to reproduce

```
>>> from fcm_django.models import FCMDevice
>>> device = FCMDevice.objects.first()
>>> device._deactivate_device_on_error_result({})
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/code.py", line 90, in runcode
    exec(code, self.locals)
  File "<console>", line 1, in <module>
  File "/usr/local/lib/python3.8/site-packages/fcm_django/models.py", line 245, in _deactivate_device_on_error_result
    device = self.objects.filter(registration_id=self.registration_id)
  File "/usr/local/lib/python3.8/site-packages/django/db/models/manager.py", line 179, in __get__
    raise AttributeError("Manager isn't accessible via %s instances" % cls.__name__)
AttributeError: Manager isn't accessible via FCMDevice instances
```

**Comment by xtrinch** [comment]
Reverted latest update.

**Comment by renceInbox** [comment]
Thanks @xtrinch !! 
This is addressed in this PR #152
